scripts/spo-create-bulk-libraries/README.md

```diff
@@ -87,6 +83,66 @@ process {
 [!INCLUDE [More about PnP PowerShell](../../docfx/includes/MORE-PNPPS.md)]
 ***
 
+# [CLI for Microsoft 365 with PowerShell](#tab/cli-m365-ps)
+
+```powershell
+# Usage example: 
+#     .\Create-Bulk-Libraries.ps1 -TargetWebUrl https://contoso.sharepoint.com/sites/Intranet AmountLibraries 3
```

Comment:
I think in the example comment we are missing the `-` 🤔. Should be `-AmountLibraries`

Reply:
Aw yeah, good catch. Let me fix that!